src/app/settings/SettingsPage.tsx

```diff
@@ -81,6 +81,11 @@ const languageOptions = mapToOptions({
 // This state is outside the settings page because the settings loses its
 let languageChanged = false;
 
+const themeOptions = mapToOptions({
+  default: 'Classic DIM (Default)',
+  dark: 'Dark',
+});
```

Comment:
These should probably be localized too but we can do that later in one go when we add some more themes.

Reply:
I'll take a run at adding a handful of themes based on in-game locations as per Ben's earlier suggestion - it'd be nice to ship this with a small selection of themes